integrationos-domain/src/domain/configuration/secrets.rs

```diff
@@ -19,9 +19,12 @@ pub struct SecretsConfig {
     pub google_kms_project_id: String,
     #[envconfig(from = "GOOGLE_KMS_LOCATION_ID", default = "global")]
     pub google_kms_location_id: String,
-    #[envconfig(from = "GOOGLE_KMS_KEY_RING_ID", default = "secrets-service-local")]
+    #[envconfig(
+        from = "GOOGLE_KMS_KEY_RING_ID",
+        default = "secrets-service-development"
```

Comment:
FYI: We're also setting the default key ring id as the `secrets-service-development` as it is where most of our keys (if not all) are stored.

cc: @paulkr @peterkuczera @moekatib

Reply:
Can the default value be omitted? If the environment variable is not set the service should fail to start.

I don't think it makes sense to provide a default value as this is an open source project and the key is not accessible to others.